test failing for droplet_test after refactor breaks travis build

https://travis-ci.org/MDLlife/MDL/jobs/349165205

```

--- FAIL: TestFromString (0.00s)
    --- FAIL: TestFromString/MDL (0.00s)
	Error Trace:	droplet_test.go:124
	Error:      	Not equal: 
	            	expected: &errors.errorString{s:"can't convert 100MDL to decimal"}
	            	actual: &errors.errorString{s:"can't convert MDL to decimal"}
	            	
	            	Diff:
	            	--- Expected
	            	+++ Actual
	            	@@ -1,2 +1,2 @@
	            	-(*errors.errorString)(can't convert 100MDL to decimal)
	            	+(*errors.errorString)(can't convert MDL to decimal)
	            	 
FAIL
FAIL	github.com/MDLlife/MDL/src/util/droplet	0.006s
```

--- a/cmd/mdl/mdl.go
+++ b/cmd/mdl/mdl.go
@@ -710,30 +710,30 @@ func Run(c *Config) {
 	}
 
 	/*
-time.Sleep(5)
-tx := InitTransaction()
-_ = tx
-err, _ = d.Visor.Visor.InjectTransaction(tx)
-if err != nil {
-   log.Panic(err)
-}
-*/
+		time.Sleep(5)
+		tx := InitTransaction()
+		_ = tx
+		err, _ = d.Visor.Visor.InjectTransaction(tx)
+		if err != nil {
+		   log.Panic(err)
+		}
+	*/
 
 	/*
-//first transaction
-if c.RunMaster == true {
-   go func() {
-	   for d.Visor.Visor.Blockchain.Head().Seq() < 2 {
-		   time.Sleep(5)
-		   tx := InitTransaction()
-		   err, _ := d.Visor.Visor.InjectTransaction(tx)
-		   if err != nil {
-			   //log.Panic(err)
-		   }
-	   }
-   }()
-}
-*/
+		//first transaction
+		if c.RunMaster == true {
+		   go func() {
+			   for d.Visor.Visor.Blockchain.Head().Seq() < 2 {
+				   time.Sleep(5)
+				   tx := InitTransaction()
+				   err, _ := d.Visor.Visor.InjectTransaction(tx)
+				   if err != nil {
+					   //log.Panic(err)
+				   }
+			   }
+		   }()
+		}
+	*/
 
 	select {
 	case <-quit:

--- a/src/gui/wallet.go
+++ b/src/gui/wallet.go
@@ -12,8 +12,8 @@ import (
 	"github.com/MDLlife/MDL/src/visor"
 	"github.com/MDLlife/MDL/src/wallet"
 
-	wh "github.com/MDLlife/MDL/src/util/http" //http,json helpers
 	"github.com/MDLlife/MDL/src/util/fee"
+	wh "github.com/MDLlife/MDL/src/util/http" //http,json helpers
 )
 
 // SpendResult represents the result of spending

--- a/src/util/droplet/droplet_test.go
+++ b/src/util/droplet/droplet_test.go
@@ -72,7 +72,7 @@ func TestFromString(t *testing.T) {
 			n: 123000456,
 		},
 		{
-			s: "MDL",
+			s: "100MDL",
 			e: errors.New("can't convert 100MDL to decimal"),
 		},
 		{

